cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/TransactionPage.java

```diff
@@ -187,8 +188,38 @@ public String setCategory(String category) {
     }
 
     public String verifyTransactionDetails() {
```

Comment:
Low impact: The 2nd, 3rd, and 4th if statements will never be run.
Suggestion: Reverse the order of the if statements or remove the unnecessary ones.